diff -upr ../frr-original/daemons ./daemons
--- ../frr-original/daemons     2025-05-15 11:27:28.413900004 +0000
+++ ./daemons   2025-05-15 11:28:57.745993030 +0000
@@ -1,3 +1,53 @@
+vtysh_enable=yes
+
+zebra=yes
+zebra_options="-A 127.0.0.1 -s 90000000"
+
+bgpd=yes
+bgpd_options="-A 127.0.0.1"
+
+bfdd=yes
+bfdd_options="-A 127.0.0.1"
+
+isisd=no
+isisd_options="-A 127.0.0.1"
+
+ldpd=no
+ldpd_options="-A 127.0.0.1"
+
+nhrpd=no
+nhrpd_options="-A 127.0.0.1"
+
+ospfd=no
+ospfd_options="-A 127.0.0.1"
+
+ospf6d=no
+ospf6d_options="-A ::1"
+
+pimd=no
+pimd_options="-A 127.0.0.1"
+
+ripd=no
+ripd_options="-A 127.0.0.1"
+
+ripngd=no
+ripngd_options="-A ::1"
+
+staticd=no
+staticd_options="-A 127.0.0.1"
+
+eigrpd=no
+eigrpd_options="-A 127.0.0.1"
+
+babeld=no
+babeld_options="-A 127.0.0.1"
+
+sharpd=no
+sharpd_options="-A 127.0.0.1"
+
+pbrd=no
+pbrd_options="-A 127.0.0.1"
+
 # This file tells the frr package which daemons to start.
 #
 # Sample configurations for these daemons can be found in
@@ -41,7 +91,7 @@ pathd=no
 vtysh_enable=yes
 zebra_options="  -A 127.0.0.1 -s 90000000"
 mgmtd_options="  -A 127.0.0.1"
-bgpd_options="   -A 127.0.0.1 -M rpki"
+bgpd_options="   -A 127.0.0.1"
 ospfd_options="  -A 127.0.0.1"
 ospf6d_options=" -A ::1"
 ripd_options="   -A 127.0.0.1"
diff -upr ../frr-original/frr.conf ./frr.conf
--- ../frr-original/frr.conf    2025-05-15 11:27:28.413900004 +0000
+++ ./frr.conf  2025-05-15 11:28:25.931253387 +0000
@@ -1,6 +1,78 @@
-!hostname frr
-
-!password frr
-!enable password frr
-
-log file /var/log/frr/frr.log
+frr version 10.3
+frr defaults datacenter
+hostname $HOSTNAME
+service integrated-vtysh-config
+!
+log file /var/log/frr/bgpd.log
+log stdout
+!
+interface eth2
+exit
+!
+interface eth3
+exit
+!
+router bgp 65401
+ bgp router-id $FABRIC_IP
+ bgp bestpath as-path multipath-relax
+ neighbor fabric peer-group
+ neighbor fabric remote-as external
+ neighbor fabric description Network Fabric
+ neighbor eth2 interface peer-group fabric
+ neighbor eth3 interface peer-group fabric
+ !
+ address-family ipv4 unicast
+  maximum-paths 2
+  neighbor fabric activate
+  neighbor fabric next-hop-self
+  neighbor fabric soft-reconfiguration inbound
+  neighbor fabric route-map FABRIC_IN in
+  neighbor fabric route-map FABRIC_OUT out
+ exit-address-family
+ !
+ address-family ipv6 unicast
+  maximum-paths 2
+  neighbor fabric activate
+  neighbor fabric next-hop-self
+  neighbor fabric soft-reconfiguration inbound
+  neighbor fabric route-map FABRIC_IN in
+  neighbor fabric route-map FABRIC_OUT out
+ exit-address-family
+exit
+!
+route-map FABRIC_IN permit 05
+ match ip address prefix-list DEFAULT_V4
+exit
+!
+route-map FABRIC_IN permit 10
+ match ipv6 address prefix-list DEFAULT_V6
+exit
+!
+route-map FABRIC_OUT deny 05
+ match ip address prefix-list DEFAULT_V4
+exit
+!
+route-map FABRIC_OUT deny 10
+ match ipv6 address prefix-list DEFAULT_V6
+exit
+!
+route-map FABRIC_OUT permit 15
+ match ip address prefix-list CUSTOMER_V4
+exit
+!
+route-map FABRIC_OUT permit 20
+ match ipv6 address prefix-list CUSTOMER_V6
+exit
+!
+route-map FABRIC_OUT permit 25
+ match ipv6 address prefix-list FABRIC_V6
+exit
+!
+route-map FABRIC_OUT permit 30
+  match ip address prefix-list FABRIC_V4
+exit
+!
+ipv6 protocol bgp route-map FABRIC_IN
+!
+end
+log syslog informational
+
